package.json

```diff
@@ -18,7 +18,7 @@
     "db:up": "lerna run --stream db:up",
     "lint:all": "lerna run --parallel lint",
     "lint:fix": "lerna run --parallel lint:fix",
-    "lint:staged": "lerna run lint:staged",
+    "lint:staged": "lerna run lint:staged --concurrency 1",
```

Comment:
where can we document why this is needed??? I guess indirectly it is mentioned in my initial issue, but little things like this seem important to be documented :thinking:

Reply:
I'll put this in the PR description